### PR TITLE
Fix query rules 2

### PR DESCRIPTION
**Pull request by markuman**
##### SUMMARY
Closes: #71

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
proxysq_query_rules

### CODE REVIEW

**Comment by markuman** [comment]
@Andersson007 can you checkout and run locally?  
works for me against 2.1.1 and 1.4.15. And yeah, changelog is missing.

File: tests/integration/targets/test_proxysql_query_rules/tasks/setup_test_query_rule.yml [hunk 1]
```diff
@@ -2,9 +2,14 @@
 - name: "{{ role_name }} | {{ current_test }}  | ensure test query rule is created when we start"
   block:
 
-    - name: "{{ role_name }} | {{ current_test }}  | ensure test query rule is created in memory"
+    - name: "{{ role_name }} | {{ current_test }}  | ensure test query rule is created in memory proxysql >= 2"
+      when: PROXYSQL2
       shell: mysql -uadmin -padmin -h127.0.0.1 -P6032 -BNe"INSERT OR REPLACE INTO mysql_query_rules (username, match_pattern, destination_hostgroup, active, retries, cache_ttl, cache_empty_result, multiplex, next_query_flagIN) VALUES ('{{ test_user }}', '{{ test_match_pattern}}', '{{ test_destination_hostgroup }}', '{{ test_active }}', '{{ test_retries }}', '{{ test_cache_ttl }}', '{{ test_cache_empty_result }}', '{{ test_multiplex }}', '{{ test_next_query_flagin }}')"
 
+    - name: "{{ role_name }} | {{ current_test }}  | ensure test query rule is created in memory proxysql >= 2"
```
Comment (Andersson007) [inline tests/integration/targets/test_proxysql_query_rules/tasks/setup_test_query_rule.yml:9]:
```suggestion
    - name: "{{ role_name }} | {{ current_test }}  | ensure test query rule is created in memory proxysql < 2"
```

**Comment by Andersson007** [comment]
> @Andersson007 can you checkout and run locally?

sure:)

> works for me against 2.1.1 and 1.4.15. And yeah, changelog is missing.

should we add `bugfixes:` one?

**Comment by Andersson007** [comment]
the tests succeeded locally for me

**Comment by markuman** [comment]
@Andersson007 changelog added.  
I re-run tests against 2.x.x and 1.4.15 on a fresh vm. LGTM.

**Review by Andersson007 — APPROVED** [review]
LGTM

**Pull request merged**